osu.Game.Tests/Beatmaps/Formats/LegacyBeatmapDecoderTest.cs

```diff
@@ -261,6 +261,11 @@ public void TestDecodeOverlappingTimingPoints()
                 Assert.That(controlPoints.EffectPointAt(2500).KiaiMode, Is.False);
                 Assert.That(controlPoints.EffectPointAt(3500).KiaiMode, Is.True);
 
+                Assert.That(controlPoints.EffectPointAt(500).OmitFirstBarLine, Is.False);
```

Comment:
I believe this would be wrong for the lazer editor (where the order specified in your test *can* actually exist).